lib/WTSI/NPG/iRODS.pm

```diff
@@ -136,6 +136,15 @@ has 'baton_client' =>
    builder  => '_build_baton_client',
    predicate => 'has_baton_client');
 
+has 'single_server' =>
+  (is            => 'ro',
+   isa           => 'Bool',
+   default       => 0,
+   documentation => 'If true, connect ony a single iRODS server by avoiding ' .
```

Comment:
Minor point: I think a word is missing in this documentation string. Probably should be "..., but does not _**require**_ resource servers to be accessible"